benches/lua.rs

```diff
@@ -17,11 +17,11 @@ fn add_fields(c: &mut Criterion) {
     let key = "the key";
     let value = "this is the value";
 
-    let key_atom_native = key.into();
+    let key_atom_native = key;
```

Comment:
Nbd, but it seems like we could remove `_atom` from these names.

Reply:
Oyyy good call.